MaintenanceDashboard.Data/Interfaces/ISpendedPaddleContext.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System.Collections.Generic;
+using MaintenanceDashboard.Data.Domain;
+
+namespace MaintenanceDashboard.Data.Interfaces
+{
+    public interface ISpendedPaddleContext
+    {
+        ICollection<SpendedPaddle> GetSpendedPaddleList();
```

Comment:
Encja to nie jest context. Ta nazwa jest bardzo myląca. Przyjęło się w .necie, że `contextem` nazywamy klasy figurujące jako baza danych (w zasadzie kontext to nie jest baza danych ale uprościmy sobie na potrzeby CR czym jest Context`.

Niech klasa bazy nazywa się `Context`, zaś te interfejsy może niech nazywają się `Repository`?

`ISpendendPaddleRepository`.  

Więcej o repozytoriach: http://www.pzielinski.com/?p=281

Wokół repozytoriów toczy się wiele dyskusji. Ogólnie `DbSet` z EFa sam w sobie jest już repozytorium. Tak samo` DbContext` jest Unit Of Workiem. To czy chcesz pisać repozytoria leży w twoim guście.